### PR TITLE
Add python3-pil-imagetk in rosdep/python.yaml

### PR DESCRIPTION
**Pull request by rkoyama1623**
Please add the following dependency to the rosdep database.

## Package name:

python3-pil-imagetk

## Package Upstream Source:

https://github.com/python-pillow/Pillow

## Purpose of using this:

The [ImageTk](https://pillow.readthedocs.io/en/stable/reference/ImageTk.html#module-PIL.ImageTk) module contains support to create and modify Tkinter BitmapImage and PhotoImage objects from PIL images. `Python-imaging-imagetk` was available up to Python2 on Ubuntu 18.04, but the Python3 version was not registered, so I submitted a PR.

Distro packaging links: https://pillow.readthedocs.io/en/stable/installation/platform-support.html

## Links to Distribution Packages

- Debian: https://packages.debian.org/sid/python3-pil.imagetk
- Ubuntu: https://packages.ubuntu.com/focal/riscv64/python3-pil.imagetk
- Fedora: https://packages.fedoraproject.org/pkgs/python-pillow/python3-pillow-tk/

### CODE REVIEW

**Review by clalancette — APPROVED** [review]

File: rosdep/python.yaml [hunk 1]
```diff
@@ -7606,6 +7606,10 @@ python3-pil:
     slackpkg:
       packages: [python3-pillow]
   ubuntu: [python3-pil]
+python3-pil-imagetk:
+  debian: [python3-pil.imagetk]
+  fedora: [python3-pillow-tk]
```
Comment (cottsay) [inline rosdep/python.yaml:7611]: Would you mind adding RHEL as well?
RHEL 8: https://pkgs.org/search/?q=python3-pillow-tk
RHEL 9: https://packages.fedoraproject.org/pkgs/python-pillow/python3-pillow-tk/ (via EPEL)

```suggestion
  fedora: [python3-pillow-tk]
  rhel: [python3-pillow-tk]
```

Comment (rkoyama1623) [reply rosdep/python.yaml:7611]: Sure, I missed them. Thank you.

**Comment by rkoyama1623** [comment]
Hmmm, I couldn't figure out why the test was failing. Could you please advise me?

**Comment by cottsay** [comment]
> Could you please advise me?

This is ros/rosdistro#41142. Hang tight while we get that merged.

**Comment by rkoyama1623** [comment]
I see. Thank you.

**Review by cottsay — APPROVED** [review]

**Comment by rkoyama1623** [comment]
Thank you for your prompt merge.

**Pull request merged**